Mathlib/Data/Fintype/Prod.lean

```diff
@@ -85,14 +85,14 @@ instance Pi.infinite_of_left {ι : Sort*} {π : ι → Sort _} [∀ i, Nontrivia
 #align pi.infinite_of_left Pi.infinite_of_left
 
 /-- If at least one `π i` is infinite and the rest nonempty, the pi type of all `π` is infinite. -/
-theorem Pi.infinite_of_exists_right {ι : Type*} {π : ι → Type*} (i : ι) [Infinite <| π i]
+theorem Pi.infinite_of_exists_right {ι : Sort _} {π : ι → Sort _} (i : ι) [Infinite <| π i]
```

Comment:
```suggestion
theorem Pi.infinite_of_exists_right {ι : Sort*} {π : ι → Sort*} (i : ι) [Infinite <| π i]
```
`Sort _` is a code smell to me, as it could indicate that there is something nontrivial to be inferred on universes, that you're leaving Lean to resolve by itself. Better use `Sort*` when it works, or `Sort (max u v)` say when needed.

On Line 79 of the same file, there is the same kind of issue (a `Sort _` which only makes sense in Type, and so should be a `Type*`). I am surprised that this one did not create issues for you. Could you nevertheless still fix it?